displayio_switchround.py

```diff
@@ -19,6 +19,13 @@
 * Adafruit CircuitPython firmware for the supported boards:
   https://github.com/adafruit/circuitpython/releases
 
+Inheritance
+-----------
+
+.. inheritance-diagram:: adafruit_displayio_layout.widgets.switch_round
```

Comment:
Should not this be .. inheritance-diagram:: displayio_switchround?
when I did this I got the following results.
![image](https://user-images.githubusercontent.com/34255413/120698049-8158c780-c47c-11eb-89ff-8eced3aad872.png)